sklearn/manifold/_mds.py

```diff
@@ -272,6 +274,71 @@ def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
         return best_pos, best_stress
 
 
+def svd_scaler(dissimilarities, n_components=2):
+    """
+    Computes multidimensional scaling using SVD algorithm
+
+    Parameters
+    ----------
+    dissimilarities : ndarray, shape (n_samples, n_samples)
+        Pairwise dissimilarities between the points. Must be euclidean.
+    n_components : int, optional, default: 2
+        Number of dimension in which to immerse the dissimilarities.
+
+    Returns
+    ----------
+    X : ndarray, shape (n_samples, n_components)
+        Coordinates of the points in a ``n_components``-space.
+
+    stress : float
+        The final value of the stress (sum of squared distance of the
+        disparities and the distances for all constrained points).
+
+    References
+    ----------
+    "An Introduction to MDS" Florian Wickelmaier
+    Sound Quality Research Unit, Aalborg University, Denmark (2003)
+
+    "Multidimensional Scaling" Chapman Hall
+    2nd edition, Boca Raton (2001)
+
```

Comment:
I'm quite ambivalent about citing a $300+ springer book.

Can we find an alternative? else I guess the other ref would be enough

Reply:
I found 'core' [paper](https://link.springer.com/article/10.1007%2FBF01896809) of that citation, but it's still paid, I've changed to it anyway. Is that ok?